server/lorax_server/utils/adapter.py

```diff
@@ -90,8 +90,8 @@ def check_architectures(model_id: str, adapter_id: str, adapter_config: "Adapter
             # Avoid execuation latency caused by the network connection retrying for AutoConfig.from_pretrained(None)
             return
 
-        expected_config = AutoConfig.from_pretrained(model_id)
-        model_config = AutoConfig.from_pretrained(adapter_config.base_model_name_or_path)
+        expected_config = AutoConfig.from_pretrained(model_id, trust_remote_code=True)
```

Comment:
Hey @thincal, the only concern I have is setting this to `True` by default exposes users to potential security vulnerabilities. We have a flag that users can provide for `--trust-remote-code` already. Can you plumb that through to this location so we don't need to default to setting it to True?

Reply:
it's reasonable, fixed.